source/_integrations/androidtv_remote.markdown

```diff
@@ -213,7 +222,7 @@ Other:
 
 {% enddetails %}
 
-If `activity` is specified in `remote.turn_on` it will open the specified URL in the associated app. See [Launching apps section](#launching-apps).
+If `activity` is specified in `remote.turn_on` it will open the specified URL or package name. See [Launching apps section](#launching-apps).
```

Comment:
```suggestion
If `activity` is specified in `remote.turn_on` it will open the specified URL or the application with the given package name. See [Launching apps section](#launching-apps).
```